utils/docker/0001-travis-fix-travisci_build_coverity_scan.sh.patch

```diff
@@ -22,6 +22,6 @@ index ad9d4afcf..562b08bcc 100644
 +  echo -e "\033[33;1mCoverity Scan upload failed: $response.\033[0m"
    exit 1
  fi
--- 
+--
```

Comment:
Again, this file is an output of another tool, so we should not modify such files.
I think you should blacklist *.patch files.